Sources/UtilityBeltUITesting/Extensions/ProcessInfo+LaunchArguments.swift

```diff
@@ -0,0 +1,12 @@
+// Copyright © 2021 SpotHero, Inc. All rights reserved.
+
+import Foundation
+
+public extension ProcessInfo {
+    /// Determines whether or not the arguments for the process contain the specific launch argument.
+    /// - Parameter argument: The argument in question.
+    /// - Returns: Whether or not the arguments for the process contain the specific launch argument.
+    static func containsLaunchArgument(_ argument: LaunchArgument) -> Bool {
+        return self.processInfo.arguments.contains(argument.formattedKey)
```

Comment:
This may need to be a bit more complex. For instance, if I added the following launch argument: `.argument(key: "env", value: "testing")`

and then called this method like so it would return true:
```swift
processInfo.containsLaunchArgument(.custom("testing"))
```

 Instead, having a property that maps the array of launch argument strings back into `LaunchArguments` may be best. `LaunchArgument` will need to become `Equatable`, but after that, you could change this to `return self.mappedArray.contains(argument)`

Reply:
@khaptonstall So, this is gonna be quite a bit to do. There's complexity around parsing the different types of arguments out of the arguments array, which will also need some associated testing. It is feeling like this should be done on a different card. I'd prefer we create a new card to be able to distinguish these new types of items outside of feature flags.

Reply:
Yeah, if that adds too much complexity/time and becomes a blocker, we can fast-follow with a separate ticket that gets queued up in this next sprint. I imagine this will roughly be something like:
```swift
var mappedArguments: [LaunchArgument] = []
while !argumentsArr.isEmpty {
    let firstArg = argumentsArr.popFirst()
    if firstArg.hasPrefix("--"), let value = argumentsArray.popFirst() {
        mappedArguments.append(.argument(key: firstArg, value: value)
    } //etc
}

```

Reply:
I've created a card. https://spothero.atlassian.net/browse/IOS-2719